docs/installation.md

```diff
@@ -0,0 +1,44 @@
+# Installing Pop!_OS
+
+The Pop!_OS installer guides the user through setup of basic system settings:
+
+- Locale and keyboard language
+- Hard drive selection and partitioning (advanced)
+- User account creation
+- System disk encryption
+
+## Installation
+
+1. Choose a language. 
+
+![Choose Language](images/installation/choose-language.png)
+
+2. Select a keyboard language layout.
+
+![Keyboard Language](images/installation/keyboard-language.png)
+
+3. Choose a clean or custom installation. <!-- See Using [GParted Custom (Advanced)](advanced-installation.md) options. -->
```

Comment:
How do I know which one I want to select?

Reply:
added clarity